climakitae/util/utils.py

```diff
@@ -685,3 +686,52 @@ def convert_to_local_time(data, selections):  # , lat, lon) -> xr.Dataset:
     selections.time_slice = (start, end)
 
     return sliced_data
+
+
+def batch_select(points):
+    """
+    Conducts batch mode analysis on a series of points for a given metric.
+    """
+    # Retrieve data for the entire service area
+    selections = Select()
+    selections.data_type = "Gridded"
+    selections.area_subset = "none"
+    selections.cached_area = ["entire domain"]
```

Comment:
This is the maximum spanning subset that a user can select from. Now, this is not ideal from a time/space complexity standpoint, but it will work.

32 stations = ~44 min. when calculating heat index (which is typically 8 min. for 1 station).

Future iterations of batch mode can address this with different architecture.

Reply:
**Note:** Reformatting code to create 32 selection objects, retrieving data, and combining the arrays together to load causes kernel to crash. This current PR may be the best approach so far, even if its runtime can be excessive.